src/test/java/AttendanceTimeFactoryTest.java

```diff
@@ -0,0 +1,18 @@
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+class AttendanceTimeFactoryTest {
+
+    @Test
+    void test1() {
+        String nickname = "빙티";
+        DateTimeFormatter dateTimeFormatter1 =
+                DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm");
+        LocalDateTime attendanceTime = LocalDateTime.parse("2024-12-06 12:02", dateTimeFormatter1);
+
+        System.out.println(attendanceTime.toString());
+    }
+
+}
```

Comment:
```suggestion

class AttendanceTimeFactoryTest {

    @Test
    void test1() {
        String nickname = "빙티";
        DateTimeFormatter dateTimeFormatter1 =
                DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm");
        LocalDateTime attendanceTime = LocalDateTime.parse("2024-12-06 12:02", dateTimeFormatter1);

        System.out.println(attendanceTime.toString());
    }
}
```

- 여기도 개행 컨밴션을 지켜봅시다
- 추가로 전체적으로 단위테스트가 누락된 곳이 많이 있는 것 같아요. 추가해볼까요?

Reply:
넵!